minio/minio
# Fix dynamic help vars for sub-commands

## Description
The fix in #7646 introduced a regression which
was left unnoticed, the fix didn't work for
sub-commands unfortunately. This fixes it
by moving v1.21.0 version of the minio/cli
package.

## Motivation and Context
Fixes #7924

## How to test this PR?
Just download the PR and build it check the output of `minio gateway s3 --help` v/s the released version.

## Types of changes

## Checklist:
- [x] Fixes a regression introduced from #7646
